.github/tests/forks/fork12.yml

```diff
@@ -5,10 +5,7 @@ args:
   verbosity: debug
 
   # https://hub.docker.com/repository/docker/leovct/zkevm-contracts/tags?name=fork.12
-  zkevm_contracts_image: leovct/zkevm-contracts:v9.0.0-rc.6-pp-fork.12
+  zkevm_contracts_image: jhkimqd/zkevm-contracts:v10.0.0-rc.6-fork.12
```

Comment:
bump of the fork12 configs to make sure we're aligned with the defaults of the package